src/main/java/com/squareup/javapoet/FieldSpec.java

```diff
@@ -89,6 +89,15 @@ public static Builder builder(Type type, String name, Modifier... modifiers) {
     return builder(TypeName.get(type), name, modifiers);
   }
 
+  public Builder toBuilder() {
+    Builder builder = new Builder(type, name);
+    builder.javadoc.add(javadoc);
+    builder.annotations.addAll(annotations);
+    builder.modifiers.addAll(modifiers);
+    builder.initializer = initializer;
```

Comment:
nit: check if this is empty, and if it is pass `null`? Otherwise you can't add an initializer on the copied instance. (I'm looking at the constructor and trying to figure out what the opposite assignment is!)